### PR TITLE
Downgrade WebError.  

### PR DESCRIPTION
**Pull request by dannon**
This fixes the interactive debugger (for now -- to be replaced w/ werkzeug eventually)

New wheels already built.

@jgoecks Should fix your issue.

### CODE REVIEW

File: lib/galaxy/dependencies/conditional-requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # These dependencies are only required when certain config options are set
 psycopg2==2.6.1
-WebError==0.11
+WebError==0.10.3
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/conditional-requirements.txt:3]: I think this line can just be removed, since WebError is already in requirements.txt

Comment (dannon) [inline lib/galaxy/dependencies/conditional-requirements.txt:3]: Hrmm.  I just updated what was there, but you're on to something here.  I think it should probably be removed from the _other_ requirements file, though.

Comment (nsoranzo) [inline lib/galaxy/dependencies/conditional-requirements.txt:3]: It was added here in commit a44b2c2170c9c326a2d5e1fec35beb44fd9834df , the in requirements.txt in 7c9c0cecdc3c31b9e86356470611ad1c7c0e87f4 , ping @natefoo.

**Comment by dannon** [comment]
We also don't need to explicitly list pygments, since that's fetched automatically as a dependency.

**Comment by nsoranzo** [comment]
I think that Pygments was mentioned to pin its version.

File: lib/galaxy/dependencies/conditional-requirements.txt (hunk 1 above)
Comment (dannon) [inline lib/galaxy/dependencies/conditional-requirements.txt:3]: I pushed the tweak removing it from the core requirements.txt and it appears to fetch and work correctly, and conditionally.

**Comment by dannon** [comment]
We shouldn't be pinning versions of dependent packages for the most part -- WebError (or whatever other first-class dependency in question) should already be defining this and double-pinning things is probably a good way to get in trouble.

File: lib/galaxy/dependencies/conditional-requirements.txt (hunk 1 above)
Comment (natefoo) [inline lib/galaxy/dependencies/conditional-requirements.txt:3]: Hrmm, I am not sure why I added it to `requirements.txt`. Seems like it should only be in `conditional-requirements.txt`.

**Comment by natefoo** [comment]
@dannon While I agree in principle, by pinning our top level dependencies, we have backed ourselves into a corner of pinning our dependencies' dependencies. Otherwise, when newer versions appear on PyPI, they will be fetched over the wheels we place on wheels.galaxyproject.org, and over time these new versions may be incompatible.

**Comment by nsoranzo** [comment]
I agree with @natefoo on this.

**Comment by dannon** [comment]
@natefoo @nsoranzo Fine, I yield :/

**Comment by dannon** [comment]
Updated to use conditional_dependencies for the pygments dependency of weberror.  They are both version locked.

**Comment by nsoranzo** [comment]
:+1: Thanks Dannon!

**Pull request merged**